### PR TITLE
background colors updating

### PR DESCRIPTION
**Pull request by trajce-bu**

### CODE REVIEW

File: ViewControllers/ViewController.swift [hunk 1]
```diff
@@ -40,13 +40,14 @@ class ViewController: UIViewController {
...
-		self.navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor:UIColor.black]
+		self.navigationController?.navigationBar.tintColor = .white
+        self.navigationController?.navigationBar.barTintColor = .systemBlue
+		self.navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor:UIColor.white]
 	}
 	
 	func authorizeWith(account: String, onCompletion:@escaping(Bool)->()) {
 		authorizedAccount = account
-		let apiKey = "YOUR_API_KEY"
+		let apiKey = "5d59cb10-66cb-405b-ab54-b4d48132f383"
```
Comment (dusan-juranovic) [inline ViewControllers/ViewController.swift]: Revert this change, since the drone repo (public) should not contain credentials.

**Review by dusan-juranovic — CHANGES_REQUESTED** [review]

File: ViewControllers/ViewController.swift (hunk 1 above)
Comment (trajce-bu) [reply ViewControllers/ViewController.swift]: updated

File: ViewControllers/ViewController.swift [hunk 2]
```diff
@@ -45,9 +45,9 @@ class ViewController: UIViewController {
 		self.navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor:UIColor.white]
 	}
 	
-	func authorizeWith(account: String, onCompletion:@escaping(Bool)->()) {
+	func authorizeWith(account: String, onComp
```
Comment (dusan-juranovic) [inline ViewControllers/ViewController.swift]: What happened here?

**Review by dusan-juranovic — APPROVED** [review]
